lib/gc/Transforms/Pipeline.cpp

```diff
@@ -128,7 +128,7 @@ void populateCPURuntimePasses(mlir::OpPassManager &pm) {
   pm.addPass(createForallToParallelLoopPass());
   pm.addPass(createParallelLoopFusionPass());
   pm.addPass(createLoopInvariantCodeMotionPass());
-  pm.addPass(createConvertMemRefToCPURuntime());
+  pm.addNestedPass<func::FuncOp>(createConvertMemRefToCPURuntime());
```

Comment:
can we merge this now?  bench tool has an issue for some mlir, and it needs this change.